### PR TITLE
updated sql exercise chapter and create a new sql orientation chapter

### PR DESCRIPTION
**Pull request by kimberly-bird**
@JoeShep @stevebrownlee 

I've added a new SQL intro chapter and updated the Music History exercise (based on the C# course content).

Let me know what you think.

### CODE REVIEW

**Comment by stevebrownlee** [comment]
Thanks for doing this @kimberly-bird 

👍

File: orientation/FND_17_SQL.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+#### For Windows
+
+Go to [SQLite Download page](http://www.sqlite.org/download.html) and download the precompiled binaries for your machine. You will need to download `sqlite-shell-win32-*.zip` and `sqlite-dll-win32-*.zip` zipped files.
+
+Create a folder `C:\sqlite` and unzip the files in this folder which will give you `sqlite3.def`, `sqlite3.dll` and `sqlite3.exe` files.
+
+Add `C:\sqlite` to your [PATH environment variable](http://dustindavis.me/update-windows-path-without-rebooting/) and finally go to the command prompt and issue `sqlite3` command.
+
+#### For Mac
```
Comment (JoeShep) [inline orientation/FND_17_SQL.md]: Let's get an 'official' Linux install instruction in here, too. We won't be making it mandatory in the future, but there's always someone running it, regardless

**Comment by kimberly-bird** [comment]
@stevebrownlee great! I updated with the commands - is there anything else Linux users need to do to instal sqlite3?

**Comment by kimberly-bird** [comment]
@JoeShep Linux commands worked for Jase - ok to merge?

**Comment by JoeShep** [comment]
:shipit: !

**Pull request merged**